__init__.py

```diff
@@ -34,6 +34,11 @@ def __init__(self):
 
     def _setup(self, force=False):
         if self.settings is not None and (force or self.ha is None):
+            ip = self.settings.get('host')
+            token = self.settings.get('token')
+            # Rise error when ip or token is not set
+            if not ip or not token:
+                self.speak_dialog('homeassistant.error.setup')
```

Comment:
This is a good first step but if you don't have a token or an IP the skill should terminate because it is just going to error out when a connection is attempted later on